### PR TITLE
feat: #1810 logging inject commit

### PR DESCRIPTION
**Pull request by AmrouEmad**
## Reference

[Code review checklist](https://github.com/linz/geostore/blob/master/CODING.md#Checklist)

### CODE REVIEW

**Comment by l0b0** [comment]
The PR title becomes the title of the commit when merging. Unfortunately the PR title isn't verified by `gitlint` (that's actually a feature that would be useful to add to the linter CI job), but that means the correctly formatted title would be "feat: #1810 logging inject commit".

File: geostore/check_files_checksums/utils.py [hunk 1]
```diff
@@ -51,7 +52,12 @@ def __init__(  # pylint:disable=too-many-arguments
 
     def log_failure(self, content: JsonObject) -> None:
         self.logger.error(
-            LOG_MESSAGE_VALIDATION_COMPLETE, extra={"outcome": Outcome.FAILED, "error": content}
+            LOG_MESSAGE_VALIDATION_COMPLETE,
+            extra={
+                "outcome": Outcome.FAILED,
+                "error": content,
+                "git_commit": get_param(ParameterName.GIT_COMMIT),
```
Comment (l0b0) [inline geostore/check_files_checksums/utils.py:59]: The "git_commit" string is used in a bunch of places, so it should probably be in a shared file, probably `geostore/logging_keys.py`.

The same goes for the other keys ("outcome", "error", etc), so feel free to do this for all the keys as a separate PR (or file an issue to do it later).

File: tests/test_check_files_checksums.py [hunk 2]
```diff
@@ -362,7 +367,6 @@ def should_mark_asset_as_replaced_in_new_version() -> None:
             multihash=storage_asset_multihash,
             exists_in_staging=True,
         ):
-
```
Comment (l0b0) [inline tests/test_check_files_checksums.py:365]: Line removal probably doesn't belong in this PR.

Comment (AmrouEmad) [reply tests/test_check_files_checksums.py]: Reverted. Thanks.

File: geostore/check_files_checksums/utils.py (hunk 1 above)
Comment (AmrouEmad) [reply geostore/check_files_checksums/utils.py]: Added git_commit to logging_keys.py. Will create another PR for the rest of keys.

**Comment by billgeo** [comment]
LGTM to me. Thanks @AmrouEmad. Will leave it to @l0b0 to approve though.

**Review by l0b0 — APPROVED** [review]
